spec.md

```diff
@@ -204,15 +205,15 @@ to DANE:
   recipient domain.
 
 * Security: DANE offers an advantage against policy-lookup DoS attacks; that is,
-  while a DNSSEC-signed NX response to a DANE lookup authoritatively indicates
+  while a DNSSEC-signed NXDOMAIN response to a DANE lookup authoritatively indicates
   the lack of a DANE record, such an option to authenticate policy non-existence
   does not exist when looking up a policy over plain DNS.
 
 # Policy Semantics
 
-SMTP STS policies are distributed at the Policy Domain either through a new
+SMTP STS policies are distributed from Policy Domain's DNS either through a new
```

Comment:
Rephrased this slightly. 

(Note I'm making a few edits to your pull request in another branch, so just commenting here so you're aware of my reasoning in case you disagree.)